.github/workflows/check-benchmarks.yml

```diff
@@ -59,6 +59,12 @@ jobs:
           rustup override unset
           rustup show
           rustup target add wasm32-unknown-unknown
+      - name: Install frame-omni-bencher
```

Comment:
To speed this up, we could do something like:

https://github.com/bkontur/runtimes/blob/ebea1aceb7a8f7ab176c2bacd90b82bfe54d505d/.github/workflows/test.yml#L88-L92